R/get_draws.R

```diff
@@ -156,7 +156,10 @@ get_draws.data.frame <- function(x,
       ) |>
       dplyr::ungroup() |>
       dplyr::rename("observed_value" = "count") |>
-      dplyr::select(-"t")
+      dplyr::select(
+        "date", "draw", "observed_value",
+        "pred_value", "total_pop"
```

Comment:
I have a weak style preference for either all one line or one name per line, depending on number of entries